README.md

```diff
@@ -1,31 +1,403 @@
-# Scaffolding
+# What is Scalprum? 
 
-## Introduction
+## “Portals on steroids!”
+
+Scalprum is a tool for building dynamic frontend UIs from a variety of different sources. Thanks to Scalprum’s dynamic nature, you can pick and choose different components that you want to pull into your UI without having to worry about rebuilding your UI each time you pull in a change. Scalprum has been built with configurability in mind - you can manage different outputs all from one configuration file, without the need to fill your code with conditionals. 
 
 Scalprum is a JavaScript micro frontend framework. It leverages [webpack](https://webpack.js.org/) 5 and its [module federation](https://webpack.js.org/concepts/module-federation/#root) features to create fast and scaleable micro frontend environments. 
 
-# Docs
 
-## @scalprum/core
+# Getting started
 
-The `@scalprum/core` package is responsible for the managing the federated modules. Behind the scenes, it uses our `@openshift/dynamic-plugin-sdk` to manage plugins (exposed webpack modules).
+### Prerequisites
 
-The core package provides an abstraction on low-level API to make it more developer friendly alongside additional features.
+- Ensure that you have [Node.js](https://nodejs.org/en/download/) installed. 
+- Environment using webpack 5 to build the final output
+- Using React frontend library
 
-## @scalprum/react-core
+### Procedure 
 
-The `@scalprum/react-core` provides React bindings for the core package. We recommend using the react-core package and its modules to build UI based on the React library.
+#### Demo environment setup 
 
-## Getting started
+The following steps outline an example webpack development setup for demo purposes. 
+You can adjust the steps to suit your own development requirements. 
 
-### Prerequisites
+1. Create a working directory for the project:
+```sh
+$ mkdir scalprum-demo && cd scalprum 
```

Comment:
I think this should be 
```sh
mkdir scalprum-demo && cd scalprum-demo
```
I don't we want the leading `$` or any other characters (like in the examples bellow) as developers can't simply copy/paste the commans.